### PR TITLE
Feature/add hierarchy for account period name

### PR DESCRIPTION
**Pull request by fivetran-avinash**
## PR Overview
**This PR will address the following Issue/Feature:** #107 

**This PR will result in the following new package version:** 13.0 (will be folded into wider release branch)

A lot of updates will be added here that will significantly revamp the package. 

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## 🚨 Breaking Changes 🚨
- Creates hierarchical strings for GL accounts (`account_display_full_name`) and accounting periods (`accounting_period_full_name`) that are then presented into the end models.

## Feature Updates 
- Added the following fields to support hierarchy data:
  - All three models: Added `subsidiary_full_name` (full hierarchical subsidiary name) and `account_display_name` (account display name).
  - `netsuite2__transaction_details`: Added `department_full_name` (full hierarchical department name), `exchange_rate` (exchange rate on accounting line), `is_eliminate` (boolean to indicate automatic elimination), `subsidiary_currency_symbol` (base currency of subsidiary), `transaction_line_amount` (net amount of transaction line).

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [NA] dbt run (if incremental models are present)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps: See Height ticket

### CODE REVIEW

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-avinash great job working through this PR!! Your validations were top notch and I feel very comfortable that these changes won't result in anything unexpected. Additionally, I appreciate the value we are adding and the collaboration you were able to do with Jared and Randy to move these forward. I look forward to additional enhancements in this batch release!

I have just a few documentation comments. Otherwise, this looks good to go!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# dbt_netsuite v0.13.0
+
+This is part of a batch release.
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md:3]: Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+## 🚨 Breaking Changes 🚨
+- Creates hierarchical strings for GL accounts (`account_display_full_name`) and accounting periods (`accounting_period_full_name`) that are then presented into the end models.
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md:6]: Updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+
+## Feature Updates 
+- Added the following fields to support hierarchy data:
+  - All three models: Added `subsidiary_full_name` (full hierarchical subsidiary name) and `account_display_name` (account display name).
+  - `netsuite2__transaction_details`: Added `department_full_name` (full hierarchical department name), `exchange_rate` (exchange rate on accounting line), `is_eliminate` (boolean to indicate automatic elimination), `subsidiary_currency_symbol` (base currency of subsidiary), `transaction_line_amount` (net amount of transaction line).
+
+## Contributors
+- [@jmongerlyra](https://github.com/jmongerlyra) ([PR #107](https://github.com/fivetran/dbt_netsuite/pull/107))
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md:14]: Added.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
+# dbt_netsuite v0.13.0
+
+This is part of a batch release.
+
+## 🚨 Breaking Changes 🚨
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md:5]: It does look as if none of the current field logic is being modified so will tweak this to Feature Updates.

**Pull request merged**